frontend/src/components/seo.tsx

```diff
@@ -75,8 +75,9 @@ const SEO: FC<Props> = ({ description, lang, meta = [], title }) => {
         },
       ]}
     >
+      <link rel="preconnect" href="https://fonts.gstatic.com" />
       <link
-        href="https://fonts.googleapis.com/css?family=Montserrat:500,700|Roboto&display=swap"
+        href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
```

Comment:
Thanks for fixing this! Are we sure we need all of these font weights?

CC: @hthomas-czi @seve 

Thanks!

Reply:
We don't need all of those. 400, 500, 600, and 700 is sufficient.

Reply:
Just for reference, those correspond to Regular, Medium, Semibold, and Bold, respectively.